frontend/src/components/common/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from 'common/types/types';
+import { useState } from 'react';
+
+import styles from '../../uam/components/group-creation/group-creation.module.scss';
```

Comment:
Seems like you have `styles` both from import and props, they'll be conflicting I guess, maybe you can style everything inside the component, without passing `styles` props